HabeebAwawda/task.py

```diff
@@ -0,0 +1,42 @@
+class Task:
+    # one
+
+    task_id = 0
```

Comment:
task_id should ideally be a class variable, which you’re using correctly. However, it's better to use class methods or static methods to encapsulate task_id manipulation to avoid direct access and modification.